src/Cocoders/CityBike/DockingStationsProvider.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Cocoders\CityBike;
+
+interface DockingStationsProvider
+{
+    /**
+     * @return DockingStation[]
+     */
+    public function getAmount();
+    public function getDockingStations();
```

Comment:
Co zwraca ta metoda? DockingStation[] czy jakiegoś array? w command widze jakieś factory która zwraca array ale nie jestem pewien czy to jest to samo.